src/commands/RequestDiscordAccessCommand.ts

```diff
@@ -0,0 +1,84 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, GuildMember } from 'discord.js';
+import { ExecuteFunction } from '../interfaces/Command';
+import * as students from '../../assets/Listado.json';
+
+function emailIsValid(email: string) {
+    const re =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
+}
+
+function padronIsValid(padron: string) {
+    const minimumPadronLength = 5;
+    const maximumPadronLength = 6;
+    return (
+        padron.length >= minimumPadronLength &&
+        padron.length <= maximumPadronLength &&
+        !isNaN(parseInt(padron))
+    );
+}
+
+function studentIsValid(email: string, padron: string): boolean {
+    for (const index in students) {
+        if (
+            students[index].includes(email) &&
+            students[index].includes(padron)
+        ) {
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
Si la propuesta anterior no funciono, puede ser porque typescript no puede inferir el tipo del archivo json. Para eso, podemos ayudarle casteandolo a un `any[]`, reemplazando `any` por algo mas concreto de ser util.
```suggestion
function studentIsValid(email: string, padron: string): boolean {
    return (students as any[]).some((student) => (
        student.includes(email) && student.includes(padron)
    ));
}
```